openapi.yaml

```diff
@@ -611,33 +625,38 @@ definitions:
             type: object
             properties:
               name:
-                $ref: '#/definitions/DeveloperResource/properties/name'
+                $ref: '#/definitions/DeveloperResource/properties/attributes/properties/name'
               website:
-                $ref: '#/definitions/DeveloperResource/properties/website'
+                $ref: '#/definitions/DeveloperResource/properties/attributes/properties/website'
   ReviewResource:
     properties:
```

Comment:
I think this resource needs a top level `links` field

Reply:
I could be wrong but I thought `links` fields were for links to other endpoints in this api which would not apply in this case. Is my understanding wrong?

Reply:
The `links` field on a resource object contains links related to the resource. We mostly just use a `self` link but I suppose there could be other values as well https://jsonapi.org/format/#document-resource-object-links